resources/acl.rb

```diff
@@ -8,8 +8,10 @@
   # As we're using the accumulator pattern we need to shove everything
   # into the root run context so each of the sections can find the parent
   with_run_context :root do
+    default_conf
     edit_resource(:template, config_file) do |new_resource|
-      cookbook 'haproxy'
+      source node['haproxy']['conf_template_source']
```

Comment:
Ok, I've moved away from node attributes (using `run_state` instead), and also dropped the library (repeated the default everywhere instead). Just waiting for the CI run to confirm it still works.